libmozevent/phabricator.py

```diff
@@ -183,3 +194,33 @@ def build_revision_url(self, build):
         Build a Phabricator frontend url for a build's revision
         """
         return "https://{}/D{}".format(self.api.hostname, build.revision_id)
+
+    def check_build_expiry(self, build):
```

Comment:
It would be better if the function name indicates what the return value represents, e.g. `is_build_still_valid` or `is_build_not_expired` (the latter has the downside to have a negation in the name).

Reply:
I renamed the method to `is_build_expired` and inverted the output so it returns `True` when the build is expired (which makes more sense and is easier to reason about)